konig-schemagen/src/main/java/io/konig/showl/WorkbookToTurtleTransformer.java

```diff
@@ -88,8 +92,20 @@ public void transform(WorkbookToTurtleRequest request) throws IOException, Sprea
 		}
 		if (!workbookFile.exists()) {
 			throw new SpreadsheetException("File not found: " + workbookFile);

```

Comment:
This exception should be thrown only if (workbookFile!=null && !workbookFile.exists)